yelp_package/dockerfiles/jammy/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Copyright 2015-2022 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG DOCKER_REGISTRY=docker-dev.yelpcorp.com/
+FROM ${DOCKER_REGISTRY}ubuntu:jammy
+
+ARG PIP_INDEX_URL=https://pypi.yelpcorp.com/simple
+ENV PIP_INDEX_URL=$PIP_INDEX_URL
+
+RUN sed -i 's/archive.ubuntu.com/us-east1.gce.archive.ubuntu.com/g' /etc/apt/sources.list
+RUN rm /etc/dpkg/dpkg.cfg.d/excludes
+RUN apt-get update && apt-get install -yq gnupg2
+
+RUN apt-get update > /dev/null && \
+    apt-get install -y --no-install-recommends software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa
+
+RUN apt-get update > /dev/null && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        build-essential \
+        debhelper \
+        gdebi-core \
+        git \
+        libffi-dev \
+        libgpgme11 \
+        libgpgme-dev \
+        libssl-dev \
+        libyaml-dev \
+        python3.8-dev \
+        python3-pip \
+        python3.8-distutils \
+        tox \
+        wget \
+        sphinx-rtd-theme-common \
+        zsh > /dev/null \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN python3.8 -m pip install --upgrade pip==20.0.2
+RUN pip3.8 install virtualenv==16.0.0
+
+RUN cd /tmp && \
+    wget http://mirrors.edge.kernel.org/ubuntu/pool/universe/d/dh-virtualenv/dh-virtualenv_1.2.2-1.1_all.deb && \
+    gdebi -n dh-virtualenv*.deb && \
+    rm dh-virtualenv_*.deb
```

Comment:
This probably isn't needed any more and can be merged back into the normal package installation stuff since the latest `dh-virtualenv` on jammy is `1.2.2` already: https://packages.ubuntu.com/jammy/dh-virtualenv

Reply:
Thanks for the comments @jvperrin 


I tried removing that highlighted bit from the dockerfile and got this error from `make itest_jammy`,  I re-added it and it went away.

```
dh: error: unable to load addon python-virtualenv: Can't locate Debian/Debhelper/Sequence/python_virtualenv.pm in @INC (you may need to install the Debian::Debhelper::Sequence::python_virtualenv module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.34.0 /usr/local/share/perl/5.34.0 /usr/lib/x86_64-linux-gnu/perl5/5.34 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.34 /usr/share/perl/5.34 /usr/local/lib/site_perl) at (eval 4) line 1.
BEGIN failed--compilation aborted at (eval 4) line 1.
```

Reply:
Did you also add `dh-virtualenv` to the list of packages installed above this when this part was removed? If so then that's strange, maybe there's something different between the versions, but I'd be surprised for such a small version difference

Reply:
I did not, just added it now and `make itest_jammy` passes, thanks!

Reply:
Awesome, that's great, no problem!